pages/components/image-input/input.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../../../styles/styles.css">
+    <link rel="icon" href="../../../assets" type="image/icon type" />
+    <link rel="stylesheet" href="./input.css">
+    <link rel="stylesheet" href="../avatar/avatar.css">
+</head>
+<body>
+    <!-- navbar -->
+   
+        <nav class="navbar flex-sb">
+            <ul class="flex-row">
+                <li><img src="../../../assets/logo.jpg" alt="logo" class="mg-l-xs"></li>
+                <li><a href="../../../index.html" class="padding-m">Home</a></li>
+                <li><a href="../avatar/avatar.html" class="padding-m">Docs</a></li>
+            </ul>
+            <ul class="social-btns flex-row">
+                <li><a href="#" class="padding-m">Linkedin</a></li>
+                <li><a href="#" class="padding-m">Gitthub</a></li>
+            </ul>
+        </nav>
+
+        <div class="wrapper">
+            <ul class="sidebar flex-column-sidebar ">
+              
+                <a href="../avatar/avatar.html" class="padding-m">Avatar</a>
+                <a href="../alert/alert.html" class="padding-m">Alert</a>
+                <a href="../badge/badge.html" class="padding-m">Badge</a>
+                <a href="../button/button.html" class="padding-m">Button</a>
+                <a href="../card/cards.html" class="padding-m">Card</a>
+                <a href="./responsive-images.html" class="padding-m">Image</a>
+                <a href="./input.html" class="padding-m">Input</a>
+                <a href="list.html" class="padding-m">List</a>
+                <a href="modal.html" class="padding-m">Modal</a>
+                <a href="other-utils.html" class="padding-m">Other Utilities</a>
+            </ul>
+      
+        
+            <div class="page-wrapper">
+                <!-- Inputs -->
+                <h2>Inputs</h2>
+                <p class="mg-t-s">Inputs are necessary for user interaction with the website. Inputs are used to take information from the user, and then save the data to the server, accordingly.</p>
+                <!-- Label Inputs -->
+                <h4 class="mg-t-s">Label Inputs</h4>
+                <p class="mg-t-s"> Label Input has a label present at the top of the input field. They can be customized to show error error-message, and change color accordingly.</p>
+
+                <div class="label-input  flex-se-column padding-s mg-l-l mg-t-s">
```

Comment:
Flex-se-column??
Use more generic name

Reply:
It means flex direction is column and justify content is space evenly.